src/main-process/parse-command-line.js

```diff
@@ -52,6 +53,7 @@ module.exports = function parseCommandLine (processArgs) {
     'When in test mode, waits until the specified time (in minutes) and kills the process (exit code: 130).'
   )
   options.alias('v', 'version').boolean('v').describe('v', 'Print the version information.')
+  options.alias('p', 'atomproject').describe('p', 'Start atom with an atomproject file.')
```

Comment:
I'd change the long-form name of this argument to just `project`, rather than `atomproject`.